products/idn/api/postman-collections.md

```diff
@@ -36,7 +36,10 @@ If there are no changes, you're up to date. If there are changes, the screen lis
 
 ## Configure your environment
 
-The SailPoint workspace provides an environment, a set of variables you can use in your requests, that you can fork and pull changes from to stay up to date the same way you can with collections.
+The SailPoint workspace provides an environment, a set of variables you can use in your requests, that you can fork and pull changes from to stay up to date the same way you can with collections. You can also fork the environment by clicking on 'Run in Postman' button.
```

Comment:
Can you add an additional short sentence here? Something like, "To import the environment into your workspace, select 'Run in Postman'."